chroot/run.go

```diff
@@ -1145,6 +1145,12 @@ func setupChrootBindMounts(spec *specs.Spec, bundlePath string) (undoBinds func(
 				return undoBinds, errors.Wrapf(err, "error mounting tmpfs to %q in mount namespace (%q, %q)", m.Destination, target, strings.Join(m.Options, ","))
 			}
 			logrus.Debugf("mounted a tmpfs to %q", target)
+		case "overlay":
+			// Mount a overlay.
+			if err := mount.Mount(m.Source, target, m.Type, strings.Join(append(m.Options, "private"), ",")); err != nil {
```

Comment:
should we raise an error when running in rootless mode or could it honor `storage.options.mount_program`?

Reply:
We should honor mount_program.  But this needs to be revealed by containers storage.

Reply:
@giuseppe for now, I am just not supporting this in Rootless mode.  The issue I have right now is we are doing the mount with runc not outside of runc, so executing fuse-overlay is difficult.  I think we should investigating fixing this in the future.

Reply:
Can/did you create an issue or jira card somewhere to track this?

Reply:
Created